README.md

```diff
@@ -32,415 +32,450 @@
-		...
-	}
-	
-	window.customElements.define("x-button", XButton, { extends: "button" })
+class MyCustomButton extends HTMLButtonElement {
+	// ...
+}
+
+window.customElements.define("my-custom-button", MyCustomButton, { extends: "button" });
 ```
-* In stead of a specific tag, the base element tag is used and enhanced with the `is` property set to the name of the customized built-in element.
+
+* Instead of a custom tag name, the tag name that corresponds to the parent class is used.
+  It is then enhanced with the `is` property, set to the name of the customized built-in element.
+
 ```html
-	<button is="x-button">Click me</button>	
+<button is="my-custom-button">Click me</x-button>
 ```
 
 ## <a name='ShadowDOM'></a>Shadow DOM
-* The shadow DOM defines a scope within which styles are applicable, together with the custom element spec, this allows for webcomponents with self contained HTML, CSS, and JS.
-*  A component's DOM is self-contained (e.g. document.querySelector() won't return nodes in the component's shadow DOM), Thus, simple id's and classes are enough for identification because they won't clash with the containing document.
-* A shadow tree can be attached to all elements that do not already host their own shadow DOM (`<texarea>`, `input`),  or where it does not make sense (`<img>`).
+* The shadow DOM defines a scope within which styles are applicable.
+  Together with the custom element spec, this allows for webcomponents with self contained HTML, CSS, and JS.
+* A component's DOM is self-contained (e.g. `document.querySelector()` will not return nodes in the component's shadow DOM).
+  Thus, simple `id`s and class names are enough for identification, because they will not clash with the containing document.
+* A shadow tree can be attached to all elements, except to those that already host their own shadow DOM (`<texarea>`, `input`), or to those for which it does not make sense (`<img>`).
 
 ```javascript
 const header = document.createElement('header');
-const shadowRoot = header.attachShadow({mode: 'open'});
+const shadowRoot = header.attachShadow({ mode: 'open' });
 const caption = document.createElement('h1');
 
 caption.appendChild(document.createTextNode('Hello Shadow DOM'));
-shadowRoot.appendChild(caption)
+shadowRoot.appendChild(caption);
 ```
 
-* A shadow tree can have a style element and even a stylesheet. These styles are local to the shadow DOM and will not bleed into the document.
+[//]: # (FIXME: What do you mean with a stylesheet, as opposed to a style element, here?)
+* A shadow tree can have a style element and even a stylesheet.
+  These styles are local to the shadow DOM and will not bleed into the document.
 
 ### <a name='Slotting'></a>Slotting
+[//]: # (FIXME: Isn't basically everything in the regular HTML part of the Light DOM?)
+[//]: # (FIXME: You are using "shadow tree" versus "Light DOM", y u no consistent?)
+[//]: # (FIXME: The bits about "default content" aren't very clear...)
```

Comment:
It's the DOM within a slot, which will be overwritten of the Light DOM provides content for that slot